var/spack/repos/builtin/packages/chai/package.py

```diff
@@ -0,0 +1,56 @@
+# Copyright 2013-2019 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+class Chai(CMakePackage, CudaPackage):
+    """
+    Copy-hiding array interface for data migration between memory spaces
+    """
+
+    homepage = "https://github.com/LLNL/CHAI"
+    git      = "https://github.com/LLNL/CHAI.git"
+
+    version('develop', branch='develop', submodules='True')
+    version('main', branch='main', submodules='True')
```

Comment:
```suggestion
    version('master', branch='main', submodules='True')
```
This will allow it to sort correctly, since main is presumably newer than all stable releases.